src/webui/www/private/scripts/dynamicTable.js

```diff
@@ -489,22 +489,26 @@ var DynamicTable = new Class({
         },
 
         selectRow : function (rowId) {
-            this.selectedRows.empty();
+            this.deselectAll();
```

Comment:
What does deselectAll() do exactly?

Reply:
It does exactly what the replaced code did, but uses an existing function.

```
        deselectAll : function () {
            this.selectedRows.empty();
        },
```